apps/framework-cli/src/cli/logger.rs

```diff
@@ -215,12 +219,30 @@ pub fn setup_logging(settings: &LoggerSettings, machine_id: &str) -> Result<(),
                 .build_log_exporter()
                 .unwrap();
 
+            let mut resource_attributes = vec![
+                KeyValue::new(SERVICE_NAME, "moose-cli"),
+                KeyValue::new("session_id", session_id.as_str()),
+                KeyValue::new("machine_id", String::from(machine_id)),
+            ];
+
+            let metric_labels = env::var("MOOSE_METRIC_LABELS")
+                .map(|encoded| general_purpose::STANDARD.decode(encoded))
+                .ok()
+                .and_then(|result| result.ok())
+                .and_then(|decoded| String::from_utf8(decoded).ok())
+                .and_then(|json_str| serde_json::from_str::<Value>(&json_str).ok())
+                .unwrap_or_else(|| serde_json::json!({}));
```

Comment:
```
            let metric_labels = env::var("MOOSE_METRIC_LABELS")
                .ok()
                .and_then(|encoded| general_purpose::STANDARD.decode(encoded).ok())
                .and_then(|json_str| serde_json::from_slice::<Value>(&json_str).ok())
                .unwrap_or_else(|| serde_json::json!({}));
```

and cache it

Reply:
also we should probably fail instead of silently ignoring the env var if it is not correct base64